src/data/set/intervals/unordered_interval.lean

```diff
@@ -60,6 +62,10 @@ interval_of_gt (lt_of_not_ge h)
 lemma interval_of_not_ge (h : ¬ b ≤ a) : [a, b] = Icc a b :=
 interval_of_lt (lt_of_not_ge h)
 
+lemma interval_eq_union : interval a b = Icc a b ∪ Icc b a := by rw [Icc_union_Icc', max_comm]; refl
```

Comment:
For consistency with the rest of the file, you should use the `[a, b]` notation here and below